Backend/StudentService.cs

```diff
@@ -33,10 +33,10 @@ public async Task UpdateStudentAsync(Student loadedStudent)
             await _studentCollection.ReplaceOneAsync(x => x.Id == loadedStudent.Id, loadedStudent);
         }
 
-        public async Task UpdateStudentTimetableAsync(Student studentToUpdate, StudyGroup studyGroup)
+        public async Task UpdateStudentTimetableAsync(Student studentToUpdate, StudentNumber studentNumber)
         {
-            studentToUpdate.Timetable = studyGroup.Timetable.Clone();
-            studentToUpdate.StudyGroup = studyGroup;
+            studentToUpdate.Timetable = studentNumber.Timetable.Clone();
```

Comment:
We have two times timetable for student? One with his changes and one from TAPI? How do you will merge them and handle them? (display, ...)

Reply:
I think its better handle Timetable for student number in our database if there will be problem with TAPI.

Reply:
OK, but it will be more complicated to manage